test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -51,6 +51,17 @@ protected RavenTestBase(ITestOutputHelper output) : base(output)
 
         private readonly object _getDocumentStoreSync = new object();
 
+        public DocumentStore GetDocumentStoreForServerOnly(X509Certificate2 certificate = null, [CallerMemberName] string caller = null)
```

Comment:
we are bloating TestBase with a method that will be used only in 1 test. Move it there.